fe/fe-core/src/main/java/org/apache/doris/system/RateLimiterMgr.java

```diff
@@ -0,0 +1,100 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.system;
+
+import com.google.common.collect.Maps;
+import com.google.common.util.concurrent.RateLimiter;
+import org.apache.doris.catalog.Catalog;
+import org.apache.doris.mysql.privilege.UserProperty;
+import org.apache.doris.thrift.TLimiterType;
+
+import java.util.Map;
+
+/**
+ * RateLimiterMgr collect all rate limiter of the system.
+ */
+public class RateLimiterMgr {
+    private RateLimiter systemQPSLimiter;
+    private RateLimiter systemStreamLoadLimiter;
+    private Map<String, RateLimiter> userToQPSLimiter;
+    private Map<String, RateLimiter> userToStreamLoadLimiter;
+
+    public RateLimiterMgr(double QPS, double streamLoadPS) {
```

Comment:
It is not recommended to pass these values through the constructor's parameters. If we subsequently add more limiter, there will be more arguments to the constructor.